### PR TITLE
Use NVMe interface for local SSD disks

### PR DESCRIPTION
**Pull request by ymmt2005**
This PR changes the interface for local SSD from scsi to NVMe for better performance.
It also increase the number of local disks from 1 to 4 because recent VM types (c2, n2, etc.)
all require 0, 4, or 8 local disks.

### CODE REVIEW

File: neco_test.go [hunk 1]
```diff
@@ -21,7 +21,6 @@ func TestNecoStartupScriptBuilder(t *testing.T) {
 
 	s := builder.Build()
 	shouldContain := []string{
-		"mkfs -t ext4 -F /dev/disk/by-id/google-local-ssd-0",
```
Comment (yokaze) [inline neco_test.go:24]: Is this line disappeared from the new implementation? I suspect there would be nvme0n1 instead.

Comment (ymmt2005) [reply neco_test.go:24]: Yes.  But I cannot find a good reason to test that.

Comment (ymmt2005) [reply neco_test.go:24]: Anyways, it works.  I am using a locally patched version :-)

Comment (yokaze) [reply neco_test.go:24]: I see, thanks for clarification.

**Review by yokaze — APPROVED** [review]
LGTM

**Pull request merged**